src/instructlab/model/evaluate.py

```diff
@@ -1,24 +1,22 @@
 # Standard
+import logging
 import os
-import subprocess
-import time
 
 # Third Party
 from instructlab.eval.evaluator import Evaluator
 from instructlab.eval.mmlu import MMLUBranchEvaluator, MMLUEvaluator
 from instructlab.eval.mt_bench import MTBenchBranchEvaluator, MTBenchEvaluator
 import click
 
+logger = logging.getLogger(__name__)
```

Comment:
@cdoern @leseb can we inherit the `ilab` logger here somehow?

Reply:
Python's logging framework automatically inherits from parent loggers. By default, a logger propagates its logs and uses settings from a parent logger. In this case it's `instructlab.model.evaluate` -> `instructlab.model` -> `instructlab` -> root logger. Because the log level defaults to `NOTSET` and there is no handler, this logger uses the defaults log level and log handler from `logging.getLogger("instructlab")`.

Reply:
Awesome - thanks!